linuxNew/README.md

```diff
@@ -120,33 +152,41 @@ rpmbuild --define "_sourcedir $(pwd)" --define "_specdir $(pwd)" \
            --define "_rpmdir $(pwd)" --rebuild *.src.rpm
 ```
 
-### Produce a Source/Binary RPM for s390x on a x86_64 Host
-
-In order to produce RPMs on an x86_64 build host for the s390x target architecture, consider this example.
-
-```shell
-spec=$(pwd)/temurin-11-jdk.spec
-mkdir temurin_s390x
-pushd temurin_s390x
-spectool --define 'vers_arch s390x' \
-           --gf ${spec}.spec
-sha256sum -c *.sha256.txt
-```
+### Building for a different architecture 
 
-Create an SRPM:
+In order to produce RPMs on an x86_64 build host for the s390x target architecture, Use the `--target` switch to `rpm-build` so as to build for a different
+archicture. Suppose the host architecture is `x86_64` and we want to build
+for target architecture `s390x`:
 
 ```shell
-rpmbuild --define 'vers_arch s390x' \
-           --define "_sourcedir $(pwd)" --define "_specdir $(pwd)" \
+$ rpmbuild --define "_sourcedir $(pwd)" --define "_specdir $(pwd)" \
            --define "_builddir $(pwd)" --define "_srcrpmdir $(pwd)" \
-           --define "_rpmdir $(pwd)" --nodeps -bs ${spec}.spec
+           --define "_rpmdir $(pwd)" --target s390x --rebuild *.src.rpm
 ```
 
-Build the binary from the SRPM:
-
-```shell
-rpmbuild --define 'vers_arch s390x' \
-           --define "_sourcedir $(pwd)" --define "_specdir $(pwd)" \
-           --define "_builddir $(pwd)" --define "_srcrpmdir $(pwd)" \
-           --define "_rpmdir $(pwd)" --target "s390x" --rebuild *.src.rpm
-```
+## Supported packages
+
+### DEB (TODO)
+
+### RPM 
+Supported JDK version 8,11,17. Supported platform x86_64, aarch64, armv7hl, ppc64le, s390x( s390x is only avaible for jdk11 and jdk11+). SRPM also available.  
+
+| Distr            | Test enabled platforms | Note |
+| ---------------- |:----------------------:|:----:|
+| amazonlinux/2    | x86_64     |                |
+| centos/7         | x86_64     |                |
+| centos/8 (switch to rocky/8)  | x86_64     ||   |
```

Comment:
Maybe delete this line, particularly since we list `rocky/8` later in the table?

Reply:
Centos/8 is still at https://packages.adoptium.net/ui/packages. Maybe update the table when centos/8 is deleted from  https://packages.adoptium.net/ui/packages.